autoscale/src/main/java/com/sequenceiq/periscope/domain/BaseAlert.java

```diff
@@ -32,6 +35,9 @@
     @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true)
     private ScalingPolicy scalingPolicy;
 
+    @Transient
+    private List<String> decommissionNodeIds;
```

Comment:
This does not belong here, even as a transient entity.
Think it is better to change the event that is being published to contain any dynamic information of this kind.

So instead of sending the BaseAlert in the SaclingEvent - send a new entity, which could internally contain the BaseAlert if required, but is more focussed on what scaling action needs to be taken. (decomissionNodeIds, newNodeCount, etc)

Reply:
yes this belong to ScalingEvent, so moved it to ScalingEvent itself.